handlers/pipeline_test.go

```diff
@@ -28,7 +28,7 @@ type mockScheduleService struct {
 	err         error
 }
 
-func (ms *mockScheduleService) SchedulePipeline(p *gaia.Pipeline, args []gaia.Argument) (*gaia.PipelineRun, error) {
+func (ms *mockScheduleService) SchedulePipeline(p *gaia.Pipeline, args []*gaia.Argument) (*gaia.PipelineRun, error) {
```

Comment:
How come this is now a pointer?

Reply:
I found some inconsistencies when and where we are using pointer or values. I just changed it now to be consistent (usually pointers).